### PR TITLE
Miscellaneous build fixups

### PR DESCRIPTION
**Pull request by tjwds**

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -35,7 +35,11 @@ jobs:
           npm run dev &
           sleep 10 &&
           wget --mirror http://localhost:3000 -P out --no-host-directories --page-requisites --adjust-extension &&
-          echo "phillyjs.com" > out/CNAME
+          wget --mirror http://localhost:3000/ad -P out --no-host-directories --page-requisites --adjust-extension
+          echo "phillyjs.com" > out/CNAME &&
+          cp public/favicon.ico out &&
+          cp public/logo.png out &&
```
Comment (JoshuaKGoldberg) [inline .github/workflows/pages.yml:41]: Curiosity: why the `&&`?

File: .github/workflows/pages.yml [hunk 2]
```diff
@@ -35,7 +35,11 @@ jobs:
           npm run dev &
           sleep 10 &&
           wget --mirror http://localhost:3000 -P out --no-host-directories --page-requisites --adjust-extension &&
-          echo "phillyjs.com" > out/CNAME
+          wget --mirror http://localhost:3000/ad -P out --no-host-directories --page-requisites --adjust-extension
+          echo "phillyjs.com" > out/CNAME &&
+          cp public/favicon.ico out &&
```
Comment (JoshuaKGoldberg) [inline .github/workflows/pages.yml:40]: Curiosity: could a `cp public/* out` work too? Do we not want that?

**Review by JoshuaKGoldberg — APPROVED** [review]
Cartoon squirrel giving a thumbs up at a computer with a GitHub sticker

File: .github/workflows/pages.yml (hunk 1 above)
Comment (tjwds) [reply .github/workflows/pages.yml:41]: Entirely laziness; it would probably be nice to format this better!

**Pull request merged**